django/db/models/deletion.py

```diff
@@ -280,6 +280,8 @@ def delete(self):
         # don't support transactions or cannot defer constraint checks until the
         # end of a transaction.
         self.sort()
+        # number of objects deleted
```

Comment:
comment is a little confusing since Counter is more than just a number - maybe "number of objects deleted for each model label"